### DISCUSSION TITLE
Port connection bricked since 12 midnight EST on 1/14/2021

### DISCUSSION THREAD

**Discussion by thbst16** [community/community#27135] [Codespaces]
<p>Port forwarding seems to have totally bricked for me in the past 24 hours without any discernable changes to my solution or what I’m doing. Tried deleting / new codespace, deleting browser cache and cookies, changing startup port, dorking with port mappings. Nothing works, infinite spinner at “Connecting to Port” for more than 24 hours. Any help would be greatly appreciated.</p>

**Comment by plisy** [comment]
<p>Could you please send over request id? I’ve been trying to find anomalies in our monitoring, but nothing stood out.</p>
<p>You can get the request id from <code>vssaas-request-id</code> header or the error page if it shows up in your case.</p>

**Comment by thbst16** [comment, marked as answer]
<p>This righted itself this weekend. I shut down all Codespace instances and the new ones that restarted no longer had issues. All good now.</p>
